server/src/main/kotlin/suwayomi/tachidesk/server/database/migration/M0027_GlobalUpdateSettings.kt

```diff
@@ -0,0 +1,28 @@
+package suwayomi.tachidesk.server.database.migration
+
+/*
+ * Copyright (C) Contributors to the Suwayomi project
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import de.neonew.exposed.migrations.helpers.SQLMigration
+import suwayomi.tachidesk.global.impl.GlobalUpdateSettingsDataClass
+import suwayomi.tachidesk.global.impl.SettingType
+import kotlin.reflect.full.memberProperties
+
+@Suppress("ClassName", "unused")
+class M0027_GlobalUpdateSettings : SQLMigration() {
+    val table = "GLOBALMETA"
+
+    override val sql = """
+        ALTER TABLE $table ADD COLUMN type varchar(256);
+        
+        UPDATE $table SET type = '${SettingType.META}';
+        
+        INSERT INTO $table (key, type, value)
+        VALUES
+            ${GlobalUpdateSettingsDataClass::class.memberProperties.joinToString(", ") { "('${it.name}', '${SettingType.GLOBAL_UPDATE}', 'true')" }};
```

Comment:
this is confusing to read, hardcode the values

Reply:
changed